e2e-tests/pitr/run

```diff
@@ -99,12 +101,17 @@ main() {
 
 	write_test_data "$cluster"
 
+	desc 'show binlog events in binlog.000005'
+	run_mysql "SHOW BINLOG EVENTS IN 'binlog.000005';" "-h $proxy -uroot -proot_password"
+
 	proxy=$(get_proxy "$cluster")
 	time_now=$(run_mysql "SELECT now();" "-h $proxy -uroot -proot_password")
-	gtid=$(run_mysql "SHOW BINLOG EVENTS IN 'binlog.000005';" "-h $proxy -uroot -proot_password" | grep Gtid | tail -c 43)
+	gtid=$(run_mysql "SHOW BINLOG EVENTS IN 'binlog.000005';" "-h $proxy -uroot -proot_password" | grep Gtid | grep -Eo "${GTID_PATTERN}")
 
-	desc 'show binlog events in binlog.000005'
-	run_mysql "SHOW BINLOG EVENTS IN 'binlog.000005';" "-h $proxy -uroot -proot_password"
+	if [[ ! ${gtid} =~ ${GTID_PATTERN} ]]; then
+		echo "Some garbage instead of legit GTID. Exiting"
```

Comment:
if we print the garbage, it'd be helpful

Reply:
we are going to do that a few lines above in desc

Reply:
moved to global variable

Reply:
I don't see where we are printing it